pkg/authorization/registry/localresourceaccessreview/rest_test.go

```diff
@@ -37,12 +39,11 @@ func (a *testAuthorizer) Authorize(attributes kauthorizer.Attributes) (allowed b
 
 	return false, "", errors.New("Unsupported")
 }
-func (a *testAuthorizer) GetAllowedSubjects(passedAttributes kauthorizer.Attributes) (sets.String, sets.String, error) {
+
+func (a *testAuthorizer) AllowedSubjects(passedAttributes kauthorizer.Attributes) ([]rbac.Subject, error) {
 	a.actualAttributes = passedAttributes
-	if len(a.err) == 0 {
-		return a.users, a.groups, nil
-	}
-	return a.users, a.groups, errors.New(a.err)
+	subjects := authorizationutil.BuildRBACSubjects(a.users.List(), a.groups.List())
```

Comment:
This is fine if the goal is to keep the diff small.  But in reality the test authorizer should just return RBAC subject directly.

Reply:
@enj I rebased/updated this PR with the changes you request. But this is the only part of your review which I'm not following.

The goal of this change was to keep the diff small as possible, plus I couldn't find any examples of usage to include RBAC subject directly. But I will be more than happy to look at this if you have any pointers of snippets.